ext/date/tests/gh10747-4.phpt

```diff
@@ -26,9 +26,17 @@ $u = unserialize($s);
 var_dump($i, str_replace(chr(0), '!', $s), $u);
 ?>
 --EXPECTF--
-object(I)#1 (11) {
+object(I)#%d (%d) {
+  ["var1":"I":private]=>
+  int(1)
+  ["var2":"I":private]=>
+  int(2)
+  ["var3":protected]=>
+  int(3)
+  ["var4":protected]=>
+  int(4)
```

Comment:
This confused me. I thought these were now added where they originally were not. GH diff is confusing sometimes.